Makefile

```diff
@@ -59,7 +59,7 @@ prepare-external: $(PACKAGES) $(NUNIT_CONSOLE)
 clean:
 	-$(MSBUILD) $(MSBUILD_FLAGS) /t:Clean
 	-rm -Rf bin/$(CONFIGURATION) bin/Build$(CONFIGURATION) bin/Test$(CONFIGURATION) bin/XAIntegration$(CONFIGURATION)
-	-rm src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config
+	-rm -f src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config
```

Comment:
Ah, looks like I misread the log output. Didn't notice the error is ignored as I am used to use -f.

    rm: src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config: No such file or directory
    make: [clean] Error 1 (ignored)